cmd/test_load.go

```diff
@@ -265,11 +265,19 @@ func loadSystemCertPool(logger logrus.FieldLogger) {
 func (lct *loadedAndConfiguredTest) buildTestRunState(
 	configToReinject lib.Options,
 ) (*lib.TestRunState, error) {
-	// This might be the full derived or just the consodlidated options
+	// This might be the full derived or just the consolidated options
 	if err := lct.initRunner.SetOptions(configToReinject); err != nil {
 		return nil, err
 	}
 
+	// Here, where we get the consolidated options, is where we check if any
```

Comment:
I added it here because it looks like a considerably safe place among all the places configuration is read/written and manipulated in general, but any suggestion will be more than welcome.

Reply:
Also, note that this will be reported despite it's a local (`k6 run`) or a remote execution (`k6 cloud`), but I guess it's fine because a script that holds `ext.loadimpact` is still a candidate of experiencing issues if we remove such support.

In any case, if we want to get real metrics, we can always account reports with this and a test run id.